Cargo.toml

```diff
@@ -39,7 +39,7 @@ unstable-debug-counters = ["future"]
 
 [dependencies]
 crossbeam-channel = "0.5.4"
-crossbeam-epoch = "0.8.2"
+crossbeam-epoch = "0.9"
```

Comment:
Can you please revert this change?

Upgrading crossbeam-epoch needs extensive testing to ensure #34 will not get worse, and I am not ready for it now.

Reply:
That's reasonable. I just wanted to bump this because the transmutes are detected by the same Miri flag (`-Zmiri-check-number-validity`) which also detects this crossbeam issue: https://github.com/crossbeam-rs/crossbeam/pull/779

Reply:
Thanks. Yeah. I know I should upgrade crossbeam-epoch soon to get those important fixes/improvements. I will try to find some time for it.